docs/content/get-started/install.md

```diff
@@ -156,3 +156,59 @@ See the [UDP and eBPF architecture]({{< ref "architecture.md#udp-and-ebpf" >}})
 
 NGINX Service Mesh does not detect changes made to the MTU in the pod at runtime.
 If adding a CNI changes the MTU of the `eth0` interface of running pods, you should re-roll the affected pods to ensure those changes take place.
+
+## Uninstall
```

Comment:
Good point. I can move the upgrade doc into the `Getting Started` section and create a separate `Uninstall` doc. That would make them easy to find.

Regarding redirects, doesn't look like there's a way to do that with hugo, but you can do it with the Netlify config. Not sure if we want to start down that path though. Don't want to end up with a bunch of redirects as the docs change. By having the install, upgrade, and uninstall docs in the same, easy-to-find location I think we can dispense with the redirect.

![Screenshot 2023-03-31 at 1 33 25 PM](https://user-images.githubusercontent.com/50845045/229224375-def3ae8b-f625-45cf-a93d-3f8ba7f02a23.png)

Reply:
I like that. Do you think it's worth adding `using nginx-meshctl` to those other docs? For consistency

Reply:
I was actually thinking about having `Install`, `Upgrade`, and `Uninstall` docs with `nginx-meshctl` and helm sections in those instead of separate docs per tool. 
```
Getting Started
    ...
    - Install
    - Upgrade
    - Uninstall
```

Alternatively could create `Install`, `Upgrade`, and `Uninstall` subsections in `Getting Started` with individual docs for `nginx-meshctl` and helm under each one like this:
```
Getting Started
    ...
    Install
        - Install using nginx-meshctl
        - Install using Helm
    Upgrade
        - Upgrade using nginx-meshctl
        - Upgrade using Helm
    Uninstall
        - Uninstall using nginx-meshctl
        - Uninstall using Helm
```
I've seen other docs for other tools/projects structured either way. Either one would be easier to navigate than the current structure.

Reply:
@ADubhlaoich Do you have any thoughts on which structure would be  better from a user perspective?

I lean towards separate docs for the two tools, but either one seems good to me.

Reply:
I'm also inclined to have separate docs for each tool and also each process (Installation, Upgrading, Uninstallation).

"Atomic" docs are easier to read and maintain because the reader and writer don't have to contend with the reading fatigue of an entire page to understand or update it.

It's also unlikely that someone installing something will have to immediately upgrade or uninstall it soon afterwards, so the extra details for processes or tools they're not interested in are just in their way when they might have limited bandwidth.

Reply:
Appreciate the feedback! I started making the suggested changes, but it's going to result in quite a large PR. It's mostly just shuffling the same content around, but I think it would be better to put that in a separate PR. @sjberman, would you be ok with approving these current changes and I'll create another PR soon with the suggested content restructure?